src/components/AdminV2/cards/NumberCard/index.jsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import {
+  Icon, Spinner, Card, Collapsible, Hyperlink,
+} from '@openedx/paragon';
+import { ArrowDownward } from '@openedx/paragon/icons';
+import { Link, useParams, useLocation } from 'react-router-dom';
+
+import { removeTrailingSlash } from '../../../../utils';
+
+const DetailAction = ({
+  slug, label, isLoading, onClick,
+}) => {
+  const location = useLocation();
+  const { actionSlug } = useParams();
+  const slugIndex = location?.pathname.lastIndexOf('/');
+  const truncatedURL = location?.pathname.substring(0, slugIndex);
+
+  return (
+    <Hyperlink
```

Comment:
Prefer using Link directly over Hyperlink to avoid unnecessary abstraction, reduce render overhead, and ensure full compatibility with React Router’s navigation and performance optimizations.